datasets/discovery/README.md

```diff
@@ -0,0 +1,20 @@
+---
+annotations_creators:
+- other
+language_creators:
+- other
+languages:
+- en
+licenses: unknown
+multilinguality:
+- monolingual
+size_categories:
+  discovery:
+  - n>1M
+  discoverysmall:
+  - 10K<n<100K
+source_datasets: []
+task_categories:
+- text-classification
+task_ids: []
+---
```

Comment:
could you add the other sections of the dataset card ?

You can leave their content with `[More Information Needed]`